pytket/extensions/qiskit/backends/aer.py

```diff
@@ -660,6 +663,71 @@ def __init__(self, n_qubits: int = 40) -> None:
         ]
 
 
+class AerDensityMatrixBackend(_AerBaseBackend):
+    """
+    Backend for running simulations on the Qiskit Aer density matrix simulator.
+
+    :param noise_model: Noise model to apply during simulation. Defaults to None.
+    :param n_qubits: The maximum number of qubits supported by the backend.
+    """
+
+    _supports_density_matrix = True
+    _supports_state = False
+    _memory = False
+    _noise_model = None
```

Comment:
type information or all these attributes would be helpful

Reply:
added in https://github.com/CQCL/pytket-qiskit/pull/380/commits/1d5fd8c8871e7393745416e91c933d2d29bbd7ad